src/org/traccar/model/Position.java

```diff
@@ -22,9 +22,11 @@
     public static final String KEY_ORIGINAL = "raw";
     public static final String KEY_INDEX = "index";
     public static final String KEY_HDOP = "hdop";
-    public static final String KEY_SATELLITES = "sat";
+    public static final String KEY_SATELLITES = "sat"; // in use
+    public static final String KEY_SATELLITES_VISIBLE = "satVisible";
```

Comment:
Typo.. it's should've been "once", not "one". To avoid polluting Position class with hundreds of keys, only ones that are used more than in one protocol are added here. If it's use only in a single place, just hardcode the key in that protocol.